galaxy_ng/app/tasks/publishing.py

```diff
@@ -3,15 +3,21 @@
 from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.utils.translation import gettext_lazy as _
-
-from pulpcore.plugin.models import Task
 from pulp_ansible.app.models import AnsibleDistribution, AnsibleRepository, CollectionVersion
 from pulp_ansible.app.tasks.collections import import_collection
+from pulpcore.plugin.models import Task
 
 from .promotion import call_copy_task, call_remove_task
 
 log = logging.getLogger(__name__)
 
+api_access_log = logging.getLogger("automated_logging")
```

Comment:
im not sure if logging config should be done in a tasking module, especially globally. This logger will always get created on module loading regardless of `GALAXY_ENABLE_API_ACCESS_LOG`,

Reply:
Apologies, should have been wrapped it in a conditional based on `GALAXY_ENABLE_API_ACCESS_LOG`